src/index.html

```diff
@@ -1,49 +1,48 @@
 <!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information. -->
 <!DOCTYPE html>
 <html>
-    <head>
-        <!-- Graph explorer depends on ACE editor -->
-        <!-- Must be inclued separetely -->
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.2.0/ace.js"></script>
 
-        <!--hello.js-->
-        <script src="../node_modules/hellojs/dist/hello.all.js"></script>
+<head>
+    <!--hello.js-->
+    <script src="../node_modules/hellojs/dist/hello.all.js"></script>
 
-        <!--Fabric-->
-        <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.4.0/css/fabric.min.css">
-        <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.4.0/css/fabric.components.min.css">
-        <script src="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.4.0/js/fabric.min.js"></script>
+    <!--Fabric-->
+    <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.4.0/css/fabric.min.css">
+    <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.4.0/css/fabric.components.min.css">
+    <script src="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-js/1.4.0/js/fabric.min.js"></script>
 
-        <!-- MWF -->
-        <link rel="stylesheet" href="https://assets.onestore.ms/cdnfiles/external/mwf/long/v1/v1.20.2/css/mwf-west-european-default.min.css">
-        <script src="https://assets.onestore.ms/cdnfiles/external/mwf/long/v1/v1.20.2/scripts/mwf-main.var.min.js"></script>
+    <!-- MWF -->
+    <link rel="stylesheet" href="https://assets.onestore.ms/cdnfiles/external/mwf/long/v1/v1.20.2/css/mwf-west-european-default.min.css">
+    <script src="https://assets.onestore.ms/cdnfiles/external/mwf/long/v1/v1.20.2/scripts/mwf-main.var.min.js"></script>
 
-        <link rel="stylesheet" href="custom.css" />
+    <link rel="stylesheet" href="custom.css" />
 
-        <!-- moment js -->
-        <script src="../node_modules/moment/min/moment-with-locales.min.js"></script>
+    <!-- moment js -->
+    <script src="../node_modules/moment/min/moment-with-locales.min.js"></script>
 
-        <!-- jQuery -->
-        <script
-            src="https://code.jquery.com/jquery-3.2.1.min.js"
-            integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-            crossorigin="anonymous"></script>
+    <!-- jQuery -->
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
```

Comment:
Yes, from my understanding, jquery is a core library that powers a lot of the other libraries in the product. e.g. fabric
I would not advice that we remove it as it may cause a problems in future